### PR TITLE
kb(grid): Add KB for transposed Grid

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: knowledge-base/grid-transposed.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+* Combine the previous two options to achieve a Form that looks like a Grid. Use the Grid's HTML markup in a [Form `FormItemsTemplate`]({%slug form-formitems-formitemstemplate%}). Optionally, remove the [built-in Form submit button by using an empty `<FormButtons>` tag]({%slug form-formitems-buttons%}).
+
+Alternatively, shape the data structure, so that it's suitable to display in a regular non-transposed Grid.
+
+The following example demonstrates all options.
+
+>caption Implement a PropertyGrid or a transposed Grid
+
+````CSHTML
+<h2>Form</h2>
```
Comment (dimodi) [inline knowledge-base/grid-transposed.md:51]:
```suggestion
@using System.Reflection

<h2>Form</h2>
```

File: knowledge-base/grid-transposed.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to transpose the data Grid component?
```
Comment (ikoevska) [inline knowledge-base/grid-transposed.md:27]: Something is off here. Should this be "the data of the Grid component" or "the Data Grid component"?

File: knowledge-base/grid-transposed.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        border-bottom-width: 0;
+    }
+
+    /* Remove unnecessary space */
+    .html-grid div.k-form-field {
+        margin-top: 0;
+    }
+</style>
+
+<h2>Readonly HTML Table</h2>
```
Comment (ikoevska) [inline knowledge-base/grid-transposed.md:163]:
```suggestion
<h2>Read-Only HTML Table</h2>
```

File: knowledge-base/grid-transposed.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+            <div class="k-grid-header-wrap">
+                <table class="k-grid-header-table k-table k-table-md">
+                    <colgroup>
+                        <col />
+                        <col />
+                    </colgroup>
+                    <thead class="k-table-thead">
+                        <tr class="k-table-row">
+                            <th class="k-header k-table-th"> Property Name </th>
+                            <th class="k-header k-table-th"> Readonly Value </th>
```
Comment (ikoevska) [inline knowledge-base/grid-transposed.md:177]:
```suggestion
                            <th class="k-header k-table-th"> Read-Only Value </th>
```

**Review by ikoevska — APPROVED** [review]

**Pull request merged**